tools/self-hosted-runner-utils/clear_offline_runners.py

```diff
@@ -1,9 +1,16 @@
 import argparse
 import os
-import re
-import sys
+import random
+import time
+from typing import List, Set
 
-from github import Github  # type: ignore[import-not-found]
+import boto3  # type: ignore[import-untyped]
+from botocore.exceptions import ClientError  # type: ignore[import-untyped]
+from github import (  # type: ignore[import-not-found]
+    Github,
+    PaginatedList,
+    SelfHostedActionsRunner,
+)
 from tqdm import tqdm  # type: ignore[import-untyped]
 
 
```

Comment:
Github won't let me propose changes to lines that didn't change :(

Adding a new argument for runner_name below would be useful as LF account has a different action runner name.

```
    parser.add_argument(
        "--runner-name",
        help="AWS Runner Name to filter for EC2 instances.",
        type=str,
        default="gh-ci-action-runner",
    )
```